src/core-plugins/CoreCommands.ts

```diff
@@ -24,9 +25,31 @@ export default class CoreCommands extends Plugin {
             }
         });
 
+        this.commands.registerCommand({
+            name: "plugins",
+            description: "List your Aliucord Plugins.",
+            options: [],
+            execute: (args, ctx) => {
+                const enabledPlugins: string[] = [];
+                const disabledPlugins: string[] = [];
+
+                for (const plugin in aliucord.pluginManager.plugins) {
+                    if (aliucord.pluginManager.isEnabled(plugin)) {
+                        enabledPlugins.push(plugin);
+                    } else {
+                        disabledPlugins.push(plugin);
+                    }
+                }
+
+                const plugins = `**Total plugins**: **${Object.keys(aliucord.pluginManager.plugins).length}**\n\n**Enabled plugins**: **${enabledPlugins.length}**\n> ${enabledPlugins.join(", ") ? enabledPlugins.join(", ") : "None."}\n**Disabled plugins**: **${disabledPlugins.length}**\n> ${disabledPlugins.join(", ") ? disabledPlugins.join(", ") : "None."}`;
+
+                ClydeUtils.sendBotMessage(ctx.channel.id, plugins);
+            }
+        });
+
         this.commands.registerCommand({
             name: "eval",
-            description: "Eval javascript",
+            description: "Evaluate JavaScript.",
```

Comment:
```suggestion
            description: "Evaluates raw JavaScript.",
```

Reply:
🤨 why is that an important distinction to make?

Reply:
I just added the "raw" because I think it flows better, but you dont have to add that
The main thing is I just don't like the statements for descriptions